build.py

```diff
@@ -0,0 +1,38 @@
+import os
+import os.path
+import shutil
+import sys
+
+
+def build(source_path, build_path, install_path, targets):
+
+    def _build():
+        # python source
+        src_py = os.path.join(source_path, "python", "tweener")
```

Comment:
Since you use `shutil.copytree` to copy the files, you don't need to put "tweener" in the `src_py` and the `dest_py`.